tests/libvmifact/factory.go

```diff
@@ -104,11 +102,15 @@ func qemuMinimumMemory() string {
 	return "1Mi"
 }
 
-func cirrosMemory() string {
+func WithSmallOSResourceMemory() {
```

Comment:
What is the meaning of with `SmallOS` resource memory? Cirros is more clear, no?